apps/vs-code-designer/src/app/commands/createNewCodeProject/createCodeProjectSteps/createLogicApp/initLogicAppCodeProjectVScode/InitCodeProject.ts

```diff
@@ -0,0 +1,370 @@
+      }
+    } else {
+      // otherwise manually edit json
+      const tasksJsonPath: string = path.join(vscodePath, tasksFileName);
+      await confirmEditJsonFile(context, tasksJsonPath, (data: ITasksJson): ITasksJson => {
+        if (!data.version) {
+          data.version = tasksVersion;
+        } else if (data.version !== tasksVersion) {
+          throw versionMismatchError;
+        }
+        data.tasks = this.insertNewTasks(data.tasks, newTasks);
+        if (this.getTaskInputs) {
+          data.inputs = this.insertNewTaskInputs(context, data.inputs, this.getTaskInputs());
+        }
+        return data;
+      });
+    }
+  }
+
+  private insertNewTasks(existingTasks: ITask[] | undefined, newTasks: ITask[]): ITask[] {
+    // tslint:disable-next-line: strict-boolean-expressions
+    existingTasks = existingTasks || [];
+    // Remove tasks that match the ones we're about to add
+    existingTasks = existingTasks.filter(
+      (t1) =>
+        !newTasks.find((t2) => {
+          if (t1.type === t2.type) {
+            switch (t1.type) {
+              case func:
+                return t1.command === t2.command;
+              case 'shell':
+              case 'process':
+                return t1.label === t2.label && t1.identifier === t2.identifier;
+              default:
+                // Not worth throwing an error for unrecognized task type
+                // Worst case the user has an extra task in their tasks.json
+                return false;
+            }
+          } else {
+            return false;
+          }
+        })
+    );
+    existingTasks.push(...newTasks);
+    return existingTasks;
+  }
+
+  private insertNewTaskInputs(context: IProjectWizardContext, existingInputs: ITaskInputs[] = [], newInputs: ITaskInputs[]): ITaskInputs[] {
+    if (context.workflowProjectType == WorkflowProjectType.Bundle) {
+      // Remove inputs that match the ones we're about to add
+      existingInputs = existingInputs.filter(
+        (t1) =>
+          !newInputs.find((t2) => {
+            if (t1.type === t2.type) {
+              switch (t1.type) {
+                case 'command':
+                  return t1.command === t2.command;
+                default:
+                  return false;
+              }
+            }
+            return false;
+          })
+      );
+      existingInputs.push(...newInputs);
+    }
+    return existingInputs;
+  }
+
+  private async writeLaunchJson(
+    context: IActionContext,
+    folder: WorkspaceFolder | undefined,
+    vscodePath: string,
+    version: FuncVersion
+  ): Promise<void> {
+    if (this.getDebugConfiguration) {
+      const newDebugConfig: DebugConfiguration = this.getDebugConfiguration(version);
+      const versionMismatchError: Error = new Error(
+        localize(
+          'versionMismatchError',
+          'The version in your {0} must be "{1}" to work with Azure Functions.',
```

Comment:
@samikay101: What did these placeholders represent? Can't remember whether I already asked this question. The string doesn't have enough info.

Reply:
{0} - tasks.json and {1} should be 2.0.0. This code snipped was reused from the original logic app project already in the codebase.